v1/models/common.py

```diff
@@ -0,0 +1,14 @@
+from typing import List
+
+from pydantic import BaseModel, validator
+
+
+class Point(BaseModel):
+    type: str
+    coordinates: List[float]
+
+    @validator('type')
+    def validate_type(cls, v):
+        if v not in ["Point", "LineString", "Polygon"]:
+            raise ValueError("Invalid geometry type")
```

Comment:
This seems confusing in a class called `Point`. Is it necessary to accept LineStrings and Polygons? Or should we call the class `Geom`?